### ISSUE TITLE
free up gulp-jspm

### ISSUE THREAD

**Issue by saelfaer** [andreasgrimm/gulp-jspm#2]
If you're not sure whether you're going to actually build this package, free up the name on npm. So someone else might actually build it.

this is an open source community where everyone can benefit from. not a domain name registrar parking domains with ads on :)

**Comment by andreasgrimm** [comment]
of course, sorry for potentially blocking this for someone else. I didn't
manage to get it going.

I just unpublished the package. Keep me up to date :)

cheers
 Andreas
